src/DBFTPlugin/Consensus/ConsensusContext.Get.cs

```diff
@@ -80,18 +101,18 @@ public UInt160 GetSender(int index)
         /// <summary>
         /// Return the expected block size
         /// </summary>
-        public int GetExpectedBlockSize()
+        public int GetExpectedBlockSize(uint i)
```

Comment:
@ZhangTao1596, maybe we can refactor code and improve this index `i` to `pOrF`, something like that, as default.

Reply:
Done.